jobs/cf-redis-broker/spec

```diff
@@ -20,7 +20,7 @@ packages:
 - redis-common
 - cf-redis-nginx
 - redis-5.0
-- redis-6.2
```

Comment:
Just a question, do we have to replace existing 6.x version with 7.x or we have to create a new package with 7.x